app/api/v1/antenna/[id]/route.ts

```diff
@@ -42,3 +42,57 @@ export async function DELETE(_: Request, context: { params: { id: string } }) {
     return NextResponse.json({ message }, { status });
   }
 }
+export async function PUT(
+  request: Request,
+  context: { params: { id: number } }
+) {
+  try {
+    if (!context.params.id) {
+      throw {
+        status: 500,
+        message: 'Internal Error: Id is missing from parameter',
```

Comment:
Follow up to previous comment: Change the error message to bad request, i.e. missing id